app/controllers/password_management_controller.rb

```diff
@@ -26,7 +22,7 @@ def send_reset_password_email
         user = @handler_result.outputs.user
         @email = @handler_result.outputs.email
         code = @handler_result.errors.first.code
-        security_log(:reset_password_failed, {user: user, email: @email, reason: code})
+        security_log(:password_reset_failed, {user: user, email: @email, reason: code})
```

Comment:
is it ok to change the name?  need to migrate date? or we are changing the definition of the event name elsewhere?